tests/test_mock_s3fs.py

```diff
@@ -0,0 +1,59 @@
+import os
+import s3fs
+import pathlib
+import pyfive
+import pytest
+import h5netcdf
+
+
+# needed by the spoofed s3 filesystem
+port = 5555
+endpoint_uri = "http://127.0.0.1:%s/" % port
+
+
+def test_s3fs_s3(s3fs_s3):
```

Comment:
@bnlawrence here is a very little test that sets up the mock S3 FS via its fixture in conftest - the object `mock_s3_filesystem` is a true S3 FS with all its attributes and methods, and you can use it to do puts and gets and what nots. It's just virtual, not a real thing :grin: